docs/chat.md

```diff
@@ -153,6 +153,30 @@ See [OCP\RichObjectStrings\Definitions](https://github.com/nextcloud/server/blob
 
 * Response: [See official OCS Share API docs](https://docs.nextcloud.com/server/latest/developer_manual/client_apis/OCS/ocs-share-api.html?highlight=sharing#create-a-new-share)
 
+## Clear chat history
+
+* Required capability: `clear-history`
+* Method: `DELETE`
+* Endpoint: `/chat/{token}`
+
+* Response:
+    - Status code:
+        + `200 OK` - When deleting was successful
+        + `202 Accepted` - When deleting was successful but Matterbridge is enabled so the message was leaked to other services
+        + `403 Forbidden` When the user is not a moderator
+        + `404 Not Found` When the conversation could not be found for the participant
+
+    - Header:
+
+        field | type | Description
+        ---|---|---
+        `X-Chat-Last-Common-Read` | int | ID of the last message read by every user that has read privacy set to public. When the user themself has it set to private the value the header is not set (only available with `chat-read-status` capability)
```

Comment:
I guess this will return the id of the system message that triggers the deletion ?

Reply:
Actually not, as the chat message is not "read" by everyone, so the value will be 0 basically all the time.